src/test/polymer/behavior-scanner_test.ts

```diff
@@ -130,4 +130,24 @@ suite('BehaviorScanner', () => {
           {name: 'getterSetter', type: undefined, readOnly: false}
         ]);
   });
+
+  const testName = 'Supports behaviors that are just arrays of other behaviors';
+  test(testName, async () => {
+    const {analyzer} = await createForDirectory(fixtureDir);
+    const analysis = await analyzer.analyze(['uses-behaviors.js']);
+    const elements = [...analysis.getFeatures({kind: 'polymer-element'})];
+    assert.deepEqual(
+        elements.map((e) => e.tagName),
+        ['uses-basic-behavior', 'uses-array-behavior']);
+
+    // Get the toplevel behaviors.
+    assert.deepEqual(
+        elements.map((e) => e.behaviorAssignments.map(ba => ba.identifier)),
+        [['BasicBehavior1'], ['ArrayOfBehaviors']]);
```

Comment:
I think I would have expected the second one to be flattened to `[BasicBehavior1, BasicBehavior2]`, since `ArrayOfBehaviors` is not a behavior. At least, the TypeScript generator assumes that this is flat, so it won't work well there. Will Analyzer let me look up what `ArrayOfBehaviors` is somehow?

Reply:
ArrayOfBehaviors _is_ a Behavior.

Reply:
Ah I see.